perceval/backends/git.py

```diff
@@ -80,11 +80,19 @@ def fetch(self, from_date=DEFAULT_DATETIME):
 
         :param from_date: obtain commits newer than a specific date
             (inclusive)
+        :param branches: consider only these branches; when None, consider
```

Comment:
Good point. Probably nonsense... I guess I should use [] instead of None, and have the current None behaviour (get all commits from all branches) implemented as [] instead. Is that ok?